Makefile.cache

```diff
@@ -75,6 +75,7 @@ SONIC_COMMON_DPKG_LIST  :=  debian/control debian/changelog debian/rules \
 SONIC_COMMON_BASE_FILES_LIST  := sonic-slave-jessie/Dockerfile.j2 sonic-slave-jessie/Dockerfile.user \
                                  sonic-slave-stretch/Dockerfile.j2 sonic-slave-stretch/Dockerfile.user \
                                  sonic-slave-buster/Dockerfile.j2 sonic-slave-buster/Dockerfile.user
+SONIC_COMMON_DOCKER_FILES_LIST := $(git ls-files files)
```

Comment:
It will have a big impact on the performance, right? The folder is changed in high frequency based on git history. Can we   reduce the scope only in sub-folders?

Reply:
Yes, agree with you.  I am working on this part as SONIC_COMMON_DOCKER_FILES_LIST list will take only the files which are common across the dockers. Changing this part is taking more time than expected.  I will submit a separate patch for this.  For the framework perspective, this change can be merged for now.